### ISSUE TITLE
Building fixturenet-eth-lighthouse on M2 fails

### ISSUE THREAD

**Issue by zramsay** [cerc-io/stack-orchestrator#298]
```
Step 14/19 : RUN cd /opt/testnet && make genesis-cl
 ---> Running in fd9677d7993e
cd cl; ./build_cl.sh
Creating testnet ...
(Note: errors of the form 'WARN: Scrypt parameters are too weak...' below can be safely ignored)
qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory
make: *** [Makefile:9: genesis-cl] Error 255
The command '/bin/sh -c cd /opt/testnet && make genesis-cl' returned a non-zero code: 2
Error running build for cerc/fixturenet-eth-lighthouse
FATAL Error: container build failed and --continue-on-error not set, exiting
```

[https://stackoverflow.com/questions/71040681/qemu-x86-64-could-not-open-lib64-ld-linux-x86-64-so-2-no-such-file-or-direc](https://stackoverflow.com/questions/71040681/qemu-x86-64-could-not-open-lib64-ld-linux-x86-64-so-2-no-such-file-or-direc)

**Comment by AFDudley** [comment]
Did the solution proposed in the link work for you?

**Comment by dboreham** [comment]
We have a big bug for ARM issues fwiw: https://github.com/cerc-io/stack-orchestrator/issues/2

**Comment by dboreham** [comment]
I think this depends on: https://github.com/cerc-io/stack-orchestrator/issues/148

(or implement the workaround for Apple ARM where the container is built for x86 and run in emulation)

**Comment by zramsay** [comment]
so, I had to enable Rosetta per: [https://collabnix.com/warning-the-requested-images-platform-linux-amd64-does-not-match-the-detected-host-platform-linux-arm64-v8/](https://collabnix.com/warning-the-requested-images-platform-linux-amd64-does-not-match-the-detected-host-platform-linux-arm64-v8/)

then add the line `FROM --platform=linux/amd64 ubuntu:20.04` to the [Dockerfile](https://github.com/cerc-io/stack-orchestrator/blob/main/app/data/container-build/cerc-fixturenet-eth-lighthouse/Dockerfile#L6)

which worked, but threw this warning:
```
Step 20/20 : ENTRYPOINT ["/opt/testnet/run.sh"]
 ---> [Warning] The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
 ---> Running in 2620c9971cdf
```
... that same warning was an error that halted `build-containers` when Rosetta wasn't enabled

**Comment by zramsay** [comment]
@AFDudley it did not until I enabled Rosetta after further digging. Ah I missed #148 that should def do the trick

**Comment by zramsay** [comment]
`[Warning] The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested` shows up all over the place now (eg building optimism), looks like instead of modifying the Dockerfile, setting these env vars (as mentioned in a stack overflow issue)
```
export DOCKER_BUILDKIT=0
export COMPOSE_DOCKER_CLI_BUILD=0
export DOCKER_DEFAULT_PLATFORM=linux/amd64
```
does the trick

**Comment by zramsay** [comment]
containers build but then deploying the MMv2 stack yields `Error response from daemon: image with reference cerc/mobymask-ui:local was found but does not match the specified platform: wanted linux/amd64, actual: linux/arm64/v8
` and S-O panics with a traceback

**Comment by zramsay** [comment]
closing in favour of #2. My workaround doesn't actually workaround + I've got what I need running in the cloud
